### PR TITLE
WildFly 27.0.0.Final

### PR DESCRIPTION
**Pull request by scottmarlow**
Signed-off-by: Scott Marlow <smarlow@redhat.com>

### CODE REVIEW

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk11.adoc [hunk 1]
```diff
@@ -0,0 +1,680 @@
...
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: jpa_core_StoredProcedureQuery
+[javatest.batch] Completed running 226 tests.
+[javatest.batch] Number of Tests Passed      = 226
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: jpa_core_annotations
```
Comment (bstansberry) [inline WildFly_27.0.0.Final/jakarta-full-platform-jdk11.adoc:269]: @scottmarlow no data here

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk11.adoc [hunk 2]
```diff
@@ -0,0 +1,680 @@
...
+----
+
+https://download.eclipse.org/jakartaee/concurrency/3.0/concurrency-tck-3.0.2.zip[Jakarta Concurrency 3.0.0 TCK]
+
+SHA-256: `22728d729f620d6a85ae903e7d1184e0a7508a4328491b785f1b4f3d7215ca93`
+
+TCK result summary:
+----
+[INFO] Running TestSuite
+[INFO] Tests run: 149, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 177.143 s - in TestSuit
```
Comment (bstansberry) [inline WildFly_27.0.0.Final/jakarta-full-platform-jdk11.adoc]: TestSuite (missing 'e' at end)

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk11.adoc (hunk 1 above)
Comment (scottmarlow) [reply WildFly_27.0.0.Final/jakarta-full-platform-jdk11.adoc:269]: nice catch!

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk11.adoc [hunk 3]
```diff
@@ -0,0 +1,680 @@
...
+
+SHA-256: `6ee953382ff965627fe20dd7e3bfce6c968ed829d611cf4990988ab54bfe8b54`
+
+Includes already certified [Eclipse Parsson 1.1.0](https://github.com/eclipse-ee4j/parsson/releases/tag/1.1.0)
+
+https://download.eclipse.org/jakartaee/mail/2.1/jakarta-mail-tck-2.1.0.zip[Jakarta Mail 2.1.0 TCK] 
+
+SHA-256: `6f02a92e0a5ef60260e65f95938cc566da2f93a3d269c3b321da0d787a3448a5`
+
+Includes already certified [Eclipse Angus Mail 1.1.0](https://github.com/eclipse-ee4j/angus-mail/releases/tag/1.0.0)
```
Comment (bstansberry) [inline WildFly_27.0.0.Final/jakarta-full-platform-jdk11.adoc]: The link is correct but the text should say 1.0.0, not 1.1.0.

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk11.adoc [hunk 4]
```diff
@@ -0,0 +1,680 @@
...
+
+SHA-256: `954fd9a3a67059ddeabe5f51462a6a3b542c94fc798094dd8c312a6a28ef2d0b`
+
+Includes already certified [Eclipse Yasson 3.0.1](https://github.com/eclipse-ee4j/yasson/releases/tag/3.0.1)
+
+https://download.eclipse.org/jakartaee/jsonp/2.1/jakarta-jsonp-tck-2.1.0.zip[Jakarta JSON Processing 2.1.0 TCK] 
+
+SHA-256: `6ee953382ff965627fe20dd7e3bfce6c968ed829d611cf4990988ab54bfe8b54`
+
+Includes already certified [Eclipse Parsson 1.1.0](https://github.com/eclipse-ee4j/parsson/releases/tag/1.1.0)
```
Comment (bstansberry) [inline WildFly_27.0.0.Final/jakarta-full-platform-jdk11.adoc]: WF 27 Final uses 1.1.1, not 1.1.0.

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc [hunk 5]
```diff
@@ -0,0 +1,680 @@
...
+----
+
+https://download.eclipse.org/jakartaee/concurrency/3.0/concurrency-tck-3.0.2.zip[Jakarta Concurrency 3.0.0 TCK]
+
+SHA-256: `22728d729f620d6a85ae903e7d1184e0a7508a4328491b785f1b4f3d7215ca93`
+
+TCK result summary:
+----
+[INFO] Running TestSuite
+[INFO] Tests run: 149, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 177.143 s - in TestSuit
```
Comment (bstansberry) [inline WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc:586]: missing 'e' in TestSuite

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc [hunk 6]
```diff
@@ -0,0 +1,680 @@
...
+
+SHA-256: `6ee953382ff965627fe20dd7e3bfce6c968ed829d611cf4990988ab54bfe8b54`
+
+Includes already certified [Eclipse Parsson 1.1.0](https://github.com/eclipse-ee4j/parsson/releases/tag/1.1.0)
+
+https://download.eclipse.org/jakartaee/mail/2.1/jakarta-mail-tck-2.1.0.zip[Jakarta Mail 2.1.0 TCK] 
+
+SHA-256: `6f02a92e0a5ef60260e65f95938cc566da2f93a3d269c3b321da0d787a3448a5`
+
+Includes already certified [Eclipse Angus Mail 1.1.0](https://github.com/eclipse-ee4j/angus-mail/releases/tag/1.0.0)
```
Comment (bstansberry) [inline WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc]: 1.0.0

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc [hunk 7]
```diff
@@ -0,0 +1,680 @@
...
+
+SHA-256: `954fd9a3a67059ddeabe5f51462a6a3b542c94fc798094dd8c312a6a28ef2d0b`
+
+Includes already certified [Eclipse Yasson 3.0.1](https://github.com/eclipse-ee4j/yasson/releases/tag/3.0.1)
+
+https://download.eclipse.org/jakartaee/jsonp/2.1/jakarta-jsonp-tck-2.1.0.zip[Jakarta JSON Processing 2.1.0 TCK] 
+
+SHA-256: `6ee953382ff965627fe20dd7e3bfce6c968ed829d611cf4990988ab54bfe8b54`
+
+Includes already certified [Eclipse Parsson 1.1.0](https://github.com/eclipse-ee4j/parsson/releases/tag/1.1.0)
```
Comment (bstansberry) [inline WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc]: 1.1.1

File: WildFly_27.0.0.Final/jakarta-web-profile-jdk11.adoc [hunk 8]
```diff
@@ -0,0 +1,325 @@
...
+
+SHA-256: `954fd9a3a67059ddeabe5f51462a6a3b542c94fc798094dd8c312a6a28ef2d0b` 
+
+Includes already certified [Eclipse Yasson 3.0.1](https://github.com/eclipse-ee4j/yasson/releases/tag/3.0.1)
+
+https://download.eclipse.org/jakartaee/jsonp/2.1/jakarta-jsonp-tck-2.1.0.zip[Jakarta JSON Processing 2.1.0 TCK]
+
+SHA-256: `6ee953382ff965627fe20dd7e3bfce6c968ed829d611cf4990988ab54bfe8b54`
+
+Includes already certified [Eclipse Parsson 1.1.0](https://github.com/eclipse-ee4j/parsson/releases/tag/1.1.0)
```
Comment (bstansberry) [inline WildFly_27.0.0.Final/jakarta-web-profile-jdk11.adoc]: 1.1.1

**Pull request merged**